### ISSUE TITLE
Possible to have FNV support?

### ISSUE THREAD

**Issue by TDarkShadow** [Mutagen-Modding/Spriggit#22]
I would like to use Spriggit to have some more verbose version control / history of my FNV mods, but I don't see it's possible at the moment.
I assume the possibility is linked until Mutagen-Modding/Mutagen/issues/22 has FNV support?

**Comment by Noggog** [comment]
Yeah, unfortunately, FNV isn't parsed in Mutagen yet, which means it's not possible for Spriggit to be used on it.   I'm currently parsing starfield, but I can perhaps knock out FNV after that since i'll be in the zone, most likely

**Comment by luckydonald** [comment]
Also fallout 3, but I heard those are very similar.

**Comment by edwardr** [comment]
+1 for FNV support
